app/Http/Controllers/BirdController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\src\Service\saveService;
+use Audubon\Entities\Bird;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use LaravelDoctrine\ORM\Facades\EntityManager;
+
+class BirdController extends Controller
+{
+    public function audubonSociety(): view
```

Comment:
Function names should be more descriptive of what they do.  In this case you're adding a sighting, so sighting or addSighting would be a better function name.